rpc/src/v1/tests/mocked/parity_set.rs

```diff
@@ -106,7 +106,24 @@ fn rpc_parity_set_min_gas_price() {
 	io.extend_with(parity_set_client(&client, &miner, &updater, &network).to_delegate());
 
 	let request = r#"{"jsonrpc": "2.0", "method": "parity_setMinGasPrice", "params":["0xcd1722f3947def4cf144679da39c4c32bdc35681"], "id": 1}"#;
-	let response = r#"{"jsonrpc":"2.0","result":false,"id":1}"#;
+	let response = r#"{"jsonrpc":"2.0","result":true,"id":1}"#;
+
+	assert_eq!(io.handle_request_sync(request), Some(response.to_owned()));
+}
+
+#[test]
+#[cfg(feature = "price-info")]
```

Comment:
I think `rpc` doesn't' have this feature so the test should be enable by default, but `TestMiner` might need a way to mock the error response.

Reply:
Still valid?

Reply:
I've removed the need for the `cfg` flag in [8f04c73](/paritytech/parity-ethereum/pull/10294/commits/8f04c73199f1314c952e6ef24c5af986973f14c2).